src/components/App.js

```diff
@@ -1,11 +1,27 @@
 import React from 'react';
+import NavBar from './NavBar.js'
+import Hero from './Hero.js'
+import HeroPicture from './HeroPicture.js'
 import StatueListContainer from './StatueListContainer.js'
+import './App.css';
+import '../index.css';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
+    <div className="app">
+      <header className="app-header">
+        <NavBar/>
       </header>
+      <div className="Hero-container">
+        <div className="row justify-content-around">
+          <div className="col-lg-6 text-center">
+          <Hero/>
+          </div>
+          <div className="col-lg-5 text-center">
+          <HeroPicture/>
+          </div>
+        </div>
+      </div>
```

Comment:
Why is the HeroPicture separate from Hero? this ` <div className="Hero-container">` should probably be the component, no?